pkg/pod/network.go

```diff
@@ -80,3 +83,37 @@ func StaticIPBondAnnotationWithInterface(
 
 	return append(annotation, bond[0])
 }
+
+// StaticIPMultiNetAnnotation defines static ip address network annotation for pod object.
+func StaticIPMultiNetAnnotation(name string, ipAddr []string) *multus.NetworkSelectionElement {
```

Comment:
Please remove this function. Use  StaticIPAnnotation instead

Reply:
This one is a [] and not sure how to use it in the for func
I will ping you

Reply:
changed to use StaticIPAnnotation